perf/lua/luafun.lua

```diff
@@ -0,0 +1,77 @@
+local fun = require("fun")
+local clock = require("clock")
+local t = require("tarantool")
```

Comment:
Minor: it is better to use `tarantool` instead of the short `t`, which is often used as an acronim for `table`. See the reason below.

Reply:
Done.